web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -400,4 +400,55 @@ describe("profile - shared", () => {
       ).not.toBeInTheDocument();
     });
   });
+
+  describe("profile error alert", () => {
```

Comment:
[dust] I think it would be helpful to have a test for the error alert appearing across the tabs, and include `owning/ oscar`